jujugui/static/gui/src/app/assets/javascripts/react-click-outside.js

```diff
@@ -21,10 +21,22 @@ function enhanceWithClickOutside(WrappedComponent) {
     componentDidMount: function componentDidMount() {
       this.__wrappedComponent = this.refs.wrappedComponent;
       document.addEventListener('click', this.handleClickOutside, true);
+      document.addEventListener('keydown', this.handleKeyDown);
```

Comment:
Now with these additions it's diverged even further from the original file - could you add a comment to this effect so that if we ever have to upgrade from the original we know what to look out for.